.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,13 @@
-## Background
-<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
+## Description of change
+Please include a description of the change. What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary.
 
-## Definition of Done
+## Testing done
+Please describe testing done to verify the changes.
```

Comment:
for some small PRs testing done and testing planned can just be accomplished through automated tests, is it necessary to have this added to every template?

Reply:
@lihanli can you point to an example of a PR where at least local acceptance testing is unnecessary?

Reply:
I think it's good to have as a to make sure that testing is thought about and executed. In the case where the change was be tested via existing automated tests or new automated tests, the submitter can say that.

Reply:
@jkassemi this is an example, https://github.com/department-of-veterans-affairs/vets-api/pull/2202, this section can be useful for more complex PRs though and we can write automated tests for small changes

Reply:
Thanks @lihanli! Sounds like @mc-usds's approach will work well in cases like that, for sure.